src/components/ContentHighlights/ZeroState/ZeroStateHighlights.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useReducer } from 'react';
+import {
+  Card, Button, Col,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import { Add } from '@edx/paragon/icons';
+import cardImage from '../data/images/ContentHighlightImage.svg';
+import ZeroStateCardImage from './ZeroStateCardImage';
+import ZeroStateCardText from './ZeroStateCardText';
+import ZeroStateCardFooter from './ZeroStateCardFooter';
+import {
+  setHighlightStepperModal,
+} from '../data/actions';
+import { initialStepperModalState } from '../data/reducer';
+import ContentHighlightStepper from '../HighlightStepper/ContentHighlightStepper';
+
+const ZeroStateHighlights = ({ cardClassName }) => {
+  /* Stepper declarations and functions - Start */
+  const [toggleModal, setToggleModal] = useState(false);
+  const [stepperModalState] = useReducer(setHighlightStepperModal, initialStepperModalState);
```

Comment:
nit: possibly another place for re-using that custom React hook with this seemingly shared stepper logic